core/LibrariesManager.vala

```diff
@@ -31,22 +30,17 @@ public class Music.LibrariesManager : GLib.Object {
      * Headless playlists are playlists that are not linked to a library.
      */
     public signal void add_headless_playlist (StaticPlaylist playlist);
-
-    public signal void library_removed (Library library);
```

Comment:
Seems odd that this signal (and the corresponding function) is not needed for a libraries manager.  Pluggable devices can add a library but do not seem to remove it when unplugged??